app/views/repos/_form.html.haml

```diff
@@ -1,3 +1,39 @@
+:javascript
+  //Add left and right arrow keys to navigate benchmark types
+  var opts = $('.result-types-form select.form-control option');
```

Comment:
We have to make sure that the document is loaded before executing the JavaScript code. You can have a look at how it is done in https://github.com/ruby-bench/ruby-bench-web/blob/3ac72884f045d112803e0370fdb79dfd9d519c16/app/views/repos/_show_chart.html.haml#L27-L30 :)

Also we should move the JavaScript code to the bottom of the file so that it gets loaded last.